migrations/20220603143022_alter_identities.up.sql

```diff
@@ -0,0 +1,10 @@
+-- alter identities id and add provider_id column 
+
+alter table auth.identities
+rename column id to provider_id;
```

Comment:
Don't forget the `{{ .index Options "Namespace" }}` and to update the filename to a more recent timestamp.